HRTool/src/main/java/app/controllers/ReportRestController.java

```diff
@@ -61,6 +58,18 @@
 	@Autowired
     private ServletContext servletContext;

```

Comment:
What is the point of this ServletContext?
This is usually a good sign that something is wrong.